### DISCUSSION TITLE
Surfshark Icon

### DISCUSSION THREAD

**Discussion by Fskum** [elrumo/macOS_Big_Sur_icons_replacements#1217] [Icon request]
I'm looking for the MacOS Surfshark icon similar to the one from iOS. The one with white background and logo in the middle. It's been in here before, but is gone now?

Is there anyone who can make a new one, or someone who can upload it again?
I would really appreciate that. Thanks in advance.

<img width="793" alt="Skærmbillede 2021-06-21 kl  20 27 39" src="https://user-images.githubusercontent.com/86257880/122810035-25d96700-d2cf-11eb-8855-8ef30ac9a216.png">

**Comment by OMGzilla-git** [comment]
Submitted. Should get approved in next day or so.

**Reply by Fskum** [reply]
Thank you very much, I look forward to using the new icon 😊

**Reply by OMGzilla-git** [reply]
Looks like they are behind on approving new submissions. You can grab it off my profile now from the waiting category if you like.

https://macosicons.com/u/OMGzilla
